contents/handbook/people/team-structure/_team_template.md

```diff
@@ -7,9 +7,6 @@ hideAnchor: true
 
 ## People
 
-- [Marius Andra (Team lead, Full Stack Engineer)](/handbook/company/team/#marius-andra-software-engineer)
```

Comment:
I put this here on purpose to give an example of how it should be formatted

Reply:
:+1: Alright, restored this _but_ with placeholder names instead of M^2. Does that look alright?

Reply:
yeah perfect